curvefs/src/mds/fs_storage.cpp

```diff
@@ -159,6 +159,33 @@ void MemoryFsStorage::GetAll(std::vector<FsInfoWrapper>* fsInfoVec) {
     }
 }
 
+FSStatusCode MemoryFsStorage::SetFsUsage(
+    const std::string& fsName, const FsUsage& fsUsage) {
+    WriteLockGuard writeLockGuard(fsUsedUsageLock_);
+    fsUsageMap_[fsName] = fsUsage;
+    return FSStatusCode::OK;
+}
+
+FSStatusCode MemoryFsStorage::GetFsUsage(
+    const std::string& fsName, FsUsage* usage, bool fromCache) {
+    WriteLockGuard readLockGuard(fsUsedUsageLock_);
+    auto it = fsUsageMap_.find(fsName);
+    if (it == fsUsageMap_.end()) {
+        fsUsageMap_[fsName] = FsUsage();
+        *usage = FsUsage();
```

Comment:
should return not found rather than a default value?

Reply:
yes